packages/engine-server/package.json

```diff
@@ -31,7 +31,7 @@
     "fluent-amqp": "^0.4.1",
     "highland": "^2.13.5",
     "id62": "^1.0.0",
-    "minio": "^7.0.18",
+    "minio": "git+https://github.com/minio/minio-js.git",
```

Comment:
We should not do this in production!
This is a workaround since I'm pretty sure we're hitting this bug https://github.com/minio/minio-js/issues/886 which has been fixed in master since december but not yet been released 😩

The issue I get with 7.0.18 is that when I have a single signature in my bucket the engine server keeps crashing with
```
/home/and/Projects/predictive_movement/predictivemovement/packages/engine-server/node_modules/minio/dist/main/xml-parsers.js:387
    xmlobj.Contents.forEach(function (content) {
                    ^

TypeError: xmlobj.Contents.forEach is not a function
```

Reply:
Idk if there's some other way to avoid this bug. Otherwise we should probably fork minio master and make our own release until they get off their butts and release a new version that fixes this issue.

Reply:
Get the same with just one signature. But I don't understand how to solve it. or rather the suggestion you had to solve it. I saw the issue ✅

Reply:
Me and Iz talked and came to the conclusion that as the data we want to store in minio is important we don't want to gamble on using an unreleased development version of it, and will instead create a workaround to avoid this bug until minio releases a new version